src/AdoNetCore.AseClient/AseConnection.cs

```diff
@@ -158,9 +163,9 @@ public new AseTransaction BeginTransaction(IsolationLevel isolationLevel = Isola
             }
 
             Open();
-            var t = new AseTransaction(this, isolationLevel);
-            t.Begin();
-            return t;
+            _transaction = new AseTransaction(this, isolationLevel);
```

Comment:
If you’re storing a reference to an IDisposable you need to dispose of it when the connection is disposed, and have a finaliser also to avoid a memory leak.

Reply:
Yeah good point

Reply:
Refer https://github.com/DataAction/AdoNetCore.AseClient/pull/31/commits/4a1e8fa6b76fdd3005b84c2ac98616e44665bf2f#diff-bd2c66437718facee056b340666da054R127